amazon_msk/assets/configuration/spec.yaml

```diff
@@ -39,3 +39,6 @@ files:
     - template: instances/openmetrics_legacy
       overrides:
         prometheus_url.hidden: true
+        prometheus_url.required: false
+        metrics.required: false
```

Comment:
Should these two defaults to false directly in the `openmetrics_legacy.yaml` template file?

Reply:
Yes good idea

Reply:
https://github.com/DataDog/integrations-core/pull/9065